Lib/test/test_asyncio/test_events.py

```diff
@@ -823,6 +823,28 @@ def test_create_server(self):
         # close server
         server.close()
 
+    def test_create_server_trsock(self):
+        proto = MyProto(self.loop)
+        f = self.loop.create_server(lambda: proto, '0.0.0.0', 0)
+        server = self.loop.run_until_complete(f)
+        self.assertEqual(len(server.sockets), 1)
+        sock = server.sockets[0]
```

Comment:
I would add a comment here that `sock` is an `asyncio.trsock.TransportSocket` instance -- this was the root of my confusion.

Reply:
Added an isinstance check.

Reply:
Thanks, good solution!